airflow_dbt_python/__version__.py

```diff
@@ -2,4 +2,4 @@
 __author__ = "Tomás Farías Santana"
 __copyright__ = "Copyright 2021 Tomás Farías Santana"
 __title__ = "airflow-dbt-python"
-__version__ = "1.0.5"
+__version__ = "1.0.6"
```

Comment:
Let's make this a minor version bump. Given the deprecations, a patch version bump is not enough.